### DISCUSSION TITLE
Package that fails to install using poetry

### DISCUSSION THREAD

**Discussion by danmichaelo** [python-poetry/poetry#3929] [Q&A]
Hi! Came across a package, [get-docker-secret](https://pypi.org/project/get-docker-secret/), which can be installed using `pip install get-docker-secret`, but fails with `poetry add get-docker-secret` with the error "Unable to find installation candidates for get-docker-secret (1.0.2)" with Python 3.8.1 and Poetry 1.1.5 on MacOS. I guess the problem is how the package is packaged, but do you see what's wrong?

**Comment by abn** [comment]
Looks like they release only eggs for [`1.0.2`](https://pypi.org/project/get-docker-secret/1.0.2/#files). However, [`1.0.1`](https://pypi.org/project/get-docker-secret/1.0.1/#files) has a tarball and should work. Maybe try `poetry add get-docker-secret@1.0.1` and raise an issue on the project perhaps.
